src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const sumObject = [];
```

Comment:
The name `sumObject` is not descriptive of what this array holds. A better name would be `stateHistory` or `statesSnapshot`, which clearly indicates that this array is a history or collection of state snapshots.